uber/configspec.ini

```diff
@@ -446,6 +446,13 @@ guest_badge     = string(default="Guest")
 one_day_badge   = string(default="One Day")
 __many__ = string
 
+[[badge_status]]
```

Comment:
I'd leave it out of core if it's not something intrinsic.  As a general rule of thumb, if something doesn't have actual logic in core that refers to ``c.WHATEVER`` then it probably shouldn't be included in the core ``configspec.ini``.  I've gradually been deleting all of the things from ``configspec.ini`` which don't meet that criteria, although there are currently plenty of exceptions I haven't gotten to yet.

Side note: I'd personally much prefer ``c.NEW`` and ``c.COMPLETED`` to ``c.NEW_STATUS`` and ``c.COMPLETED_STATUS``.  It's not a problem if the same constant ends up in two different enums, so I don't think it's necessary to append ``_STATUS`` to the end of every one of these.